src/main/java/com/spotifyapi/filter/TokenValidation.java

```diff
@@ -26,23 +25,30 @@ public class TokenValidation extends OncePerRequestFilter {
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
             throws ServletException, IOException {
 
+        if (request.getRequestURI().equals("/api/login") || request.getRequestURI().equals("/api/profile")) {
```

Comment:
OncePerRequestFilter has a specific method where you can move this logic

Reply:
I going to use that method in Filter class: shouldNotFilter. I hope it's possible, because, seems to me, everything is work with it